src/index.html

```diff
@@ -17,6 +17,10 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__minutes stopwatch__minutes--position"></div>
+      <div class="stopwatch__seconds stopwatch__seconds--position"></div>
+      <div class="stopwatch__circle"></div>
```

Comment:
Ensure that the elements have content or are used with JavaScript. If they are intended to display time, they should be filled with some default content like '00', or have some accessible labels or titles for better accessibility.